File: frontend/src/dto/MachineDTO.ts
export class MachineDTO{
    private id: number;
	private name: string;
	private model: string;
    private idUser: number;
    
    constructor(id: number, name: string, model: string, idUser: number){
        this.id = id;
        this.idUser = idUser;
        this.model = model;
        this.name = name;
    }
}

File: backend_old/MachineMicroservice/src/main/java/com/MachineMicroservice/dto/MachineDTO.java
package com.MachineMicroservice.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class MachineDTO {
	
	private int id;
	private String name;
	private String model;
	private int idUser;
}
